modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -22,7 +22,34 @@ You can also deploy to CloudHub through:
 
 == Deploying an Application
```

Comment:
I would leave the CloudHub in the title because this is specific to deploying to CloudHub. There are other deployment topics for deploying to other targets.